### PR TITLE
Documented an optional header for the Teams Shifts APIs

### PR DESCRIPTION
**Pull request by shreniksv**
Updated Graph API documents to add the information about an optional header, MS-APP-ACTS-AS, that takes Guid as a value which points to an user id. The header is required only if the Authorization token is an application token. It is not applicable for user authorization tokens.

### CODE REVIEW

**Comment by JarbasHorst** [comment]
Hello @shreniksv, should the [Schedule create or replace](https://learn.microsoft.com/en-us/graph/api/team-put-schedule?view=graph-rest-1.0) operation also contain the new header? What about BETA endpoints?

Also, to avoid merge conflicts, please merge PR [21427](https://github.com/microsoftgraph/microsoft-graph-docs/pull/21427) into your branch to include my fixes and before addressing my new comments.

File: changelog/Microsoft.Teams.Shifts.json [hunk 1]
```diff
@@ -1,112 +1,138 @@
...
+         }
+       ],
+       "Id": "c9923370-4dca-4605-9027-77a05ddfdc8b",
+       "Cloud": "prd",
+       "Version": "beta",
+       "CreatedDateTime": "2021-06-04T01:15:33.264Z",
+       "WorkloadArea": "Teamwork and communications",
+       "SubArea": "Shift management"
+     },
+     {
```
Comment (JarbasHorst) [inline changelog/Microsoft.Teams.Shifts.json:111]: Here starts the changelog for this PR.

**Comment by Jarbas-MSFT** [comment]
Hi @shreniksv, not sure if you have missed this comment. In any case, 
- should the [Schedule create or replace](https://learn.microsoft.com/en-us/graph/api/team-put-schedule?view=graph-rest-1.0) operation also contain the new header?
- should BETA endpoints also be updated or do these changes only apply to v1.0 topics?

**Comment by shreniksv** [comment]
@JarbasHorst I have updated the docs for beta API and addressed the comment on the missing doc too. Could you please review the updates? Thanks,

**Comment by Jarbas-MSFT** [comment]
Hi @Lauragra, this PR is ready for final review.

@shreniksv, thanks for your changes. I am done with the content review work for your PR now.

File: changelog/Microsoft.Teams.Shifts.json [hunk 2]
```diff
@@ -1,112 +1,148 @@
...
+       "ChangeList": [
+         {
+           "Id": "8662d342-a455-4111-a5a0-fa023070440a",
+           "ApiChange": "Header",
+           "ChangedApiName": "offerShiftRequest, openShift, openShiftChangeRequest, schedule, schedulingGroup, shift, shiftPreferences, swapShiftsChangeRequest, timeOff, timeOffReason, timeOffRequest, workforceIntegration",
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.Shifts.json:116]: The ChangedApiName and Target fields are single valued, so unfortunately, we need to create separate entries for each of these.

Comment (JarbasHorst) [reply changelog/Microsoft.Teams.Shifts.json:116]: It would have been too easy otherwise :)

**Review by Lauragra — APPROVED** [review]

**Comment by shreniksv** [comment]
@Lauragra @JarbasHorst @Jarbas-MSFT 
I don't have merge permission. Could you please merge the branch? or is this something that bot will do for us as the branch is labelled as "ready to merge"?

**Pull request merged**